offsets.py

```diff
@@ -205,6 +215,11 @@ class OffsetsFirmware_850_852:
 
     MEMCPY = 0xffffffff825a40f0
 
+    # Not available on FW >= 6.50
```

Comment:
Why do you have these comments

Reply:
Pre 6.50 you don't have the ud2 in that gadget, we can actually remove the `MOV_CR0_RSI_MOV_EAX_1_RET` gadget from FW >=6.50 and the `MOV_CR0_RSI_UD2_MOV_EAX_1_RET` from FW < 6.50. I just had them there when I was working on stuff so I could keep stuff straight, they're just the base @ ffffffff82200000 as a placeholder.

Or you can remove the conditional when the gadget is used and just use one of them for both (variable name won't match contents).

Reply:
Well please remove them